src/vulkan/pipeline.cc

```diff
@@ -185,10 +221,35 @@ Result Pipeline::CreateDescriptorSets() {
 }
 
 Result Pipeline::CreatePipelineLayout() {
+  // Put all elements of |descriptor_set_layouts_| and
+  // |empty_descriptor_set_layouts_| together. Note that their
+  // order and the order of actually used descriptor sets must
+  // exactly match.
```

Comment:
I don't understand what this means "their order and the order of actually used descriptor sets must exactly match" ?

Reply:
I removed this code to put both empty and non-empty ones together.